src/deepsparse/license.py

```diff
@@ -74,7 +75,31 @@ def add_deepsparse_license(token_or_path):
     validate_license()
 
 
-def validate_license(license_path: Optional[str] = None, print_splash: bool = True):
+def license_status(license_path: Optional[str] = None) -> Tuple[bool, str, str]:
+    """
+    Returns the status of a license as tuple of 
+    (is_exception, error/warning, splash message)
+    
+    License should be passed as a text file containing only the JWT. If no path is
+    provided the expected file path of the token will be validated. Default
+    path is ~/.config/neuralmagic/license.txt
+
+    :param license_path: file path to text file of token to check status of.
+        Default is None, expected token path will be validated
+    :return: tuple of (is_exception, error/warning, splash message)
```

Comment:
fyi, if is_exception is true then the splash will have the same value as the error message.   there's no deep reason for this but it's just for convenience because of how the c++ code is written.

i don't think it should matter but thought i would mention it.

Reply:
updated - so only warnings go in the second slot?